packages/components/Button/src/Button.test.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { Button } from './Button';
 import * as renderer from 'react-test-renderer';
+import '@types/jest';
```

Comment:
Can't you achieve the same by updating the `types` attribute in the `tsconfig.json` file?

```ts
compilerOptions: {
    types: ["node", "jest"]
}
```
Then you don't have to do this in every test file.

Reply:
We thought about this but we'd need to list every other type it requires too, right? That'd be a long list. All to ensure 'jasmine' doesn't get referenced.

What would be nice is an attribute that does the opposite of this but I don't believe it exists.

Reply:
Have you actually tried it? I'd think this `types` attribute only affect global type declaration. If jest explicitly include other modules, those types will be checked by Typescript.

Reply:
I see. Yeah that works for me locally.